src/components/NavBar/NavBarToolItem.tsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom';
```

Comment:
I don't think you need to define a component for this one 🤔 
If its purpose is only to be a `<span />` within a `<Link />`, you could just write it in `NavBarTools` instead